apps/events/tables.py

```diff
@@ -46,8 +46,11 @@ def render(self, value, record):
 
 class ParamsColumn(tables.Column):
     def render(self, value, record):
-        items = format_html_join("", "<li><strong>{}</strong>: {}</li>", value.items())
-        return format_html(f"<ul>{items}</ul>")
+        try:
+            items = format_html_join("", "<li><strong>{}</strong>: {}</li>", value.items())
+            return format_html(f"<ul>{items}</ul>")
```

Comment:
this isn't actually doing a format since it's using a f-string. Changing this line to `format_html("<ul>{}</ul>", items)` should also fix the issue.

Reply:
sweet thanks-- fyi this does make the row take up the whole screen for the summary each time unless we reformat, just flagging in case thats a deal breaker
<img width="659" alt="Screenshot 2025-02-21 at 3 32 51 PM" src="https://github.com/user-attachments/assets/51a39c2d-cd24-416a-8612-fa50176c6e70" />

Reply:
I think we should trim the value to a smaller size - maybe 100 chars

Reply:
https://github.com/dimagi/open-chat-studio/pull/1235#:~:text=%E2%80%A6-,8e849fb,-Merge%20branch%20%27smh